src/tests/ftest/util/write_host_file.py

```diff
@@ -5,10 +5,10 @@
   SPDX-License-Identifier: BSD-2-Clause-Patent
 """
 from logging import getLogger
-
 import os
 import random
 
+
 def write_host_file(hostlist, path='/tmp', slots=1):
```

Comment:
@phender Should we update the docstring to indicate hostlist could be a NodeSet? Also, if a list is passed, should we convert it to a NodeSet for logging purposes? (conserves space)

Reply:
Done